bql/planner/planner_test.go

```diff
@@ -491,6 +479,94 @@ func TestPlannerQuery(t *testing.T) {
 	}
 }
 
+func TestPlannerConstruct(t *testing.T) {
```

Comment:
Done. Since we do not know what the blank node id is going to be, I had to add a lot of logic here to match place holder blank nodes to blank nodes generated by triples.go. Let me know if looks fine.